opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -430,6 +447,8 @@ int opal_common_ucx_wpmem_create(opal_common_ucx_ctx_t *ctx, void **mem_base, si
     OBJ_CONSTRUCT(&mem->tls_key, opal_tsd_tracked_key_t);
     opal_tsd_tracked_key_set_destructor(&mem->tls_key, _mem_rec_destructor);
 
+    mem->comm = exchange_metadata;
```

Comment:
Is this used anywhere? I couldn't find where the `void*` stored in mem is used

Reply:
This comm pointer no longer is needed. Thanks for pointing it out. I removed it.

Reply:
Just pushed the code.